lib/octokit/connection.rb

```diff
@@ -175,15 +175,9 @@ def sawyer_options
       conn_opts = @connection_options
       conn_opts[:builder] = @middleware if @middleware
       conn_opts[:proxy] = @proxy if @proxy
-      if conn_opts[:ssl].nil?
-        conn_opts[:ssl] = { :verify_mode => @ssl_verify_mode } if @ssl_verify_mode
-      else 
-        if @connection_options[:ssl][:verify] == false
-          conn_opts[:ssl] = { :verify_mode => 0}
-        else
-          conn_opts[:ssl] = { :verify_mode => @ssl_verify_mode }
-        end
-      end
+      conn_opts[:ssl] ||= {}
+      conn_opts[:ssl][:verify_mode] ||= 0 if conn_opts.dig(:ssl, :verify) == false
+      conn_opts[:ssl][:verify_mode] ||= @ssl_verify_mode if @ssl_verify_mode
```

Comment:
This feels pretty compact, and kind of difficult to read.

Would you mind putting it back to the same formatting as the original?

Reply:
How about this: fe4a4185?